content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -363,6 +363,26 @@ rules:
   verbs: ["get", "list", "watch"]
 ```
 
+#### Write Access for Endpoints
+
+Kubernetes clusters created before Kubernetes v1.22 include write access to
+Endpoints in the aggregated "edit" and "admin" roles. As a mitigation for
+[CVE-2021-25740](https://github.com/kubernetes/kubernetes/issues/103675), this
+access is not part of the aggregated roles in clusters that you create using
+Kubernetes v1.22 or later.
+
+Existing clusters that have been upgraded to Kubernetes v1.22 will not be
+subject to this change. The [CVE
+announcement](https://github.com/kubernetes/kubernetes/issues/103675) includes
+guidance for restricting this access in existing clusters.
+
+If you want new clusters to retain this level of access in the aggregated roles,
+you can create the following ClusterRole:
+
+```none
+{{</* codenew file="access/endpoints-aggregated.yaml" */>}}
+```
```

Comment:
This doesn't render how you expect.
Try:
```suggestion
{{< codenew file="access/endpoints-aggregated.yaml" >}}
```

Reply:
Updated, thanks!